### DISCUSSION TITLE
Find character width of terminal

### DISCUSSION THREAD

**Discussion by jmacwhyte** [jcubic/jquery.terminal#881] [Q&A] [answered]
Hello! Is there a way to find out how many characters wide the terminal is? I see a way to get the cursor position and a way to set the number of characters, but I don't know how to find out the width if it has been set automatically.

**Comment by jcubic** [comment, marked as answer]
Yes, you can use `term.cols()`. The names cols and rows are used by Linux terminals that's why they are named like this.

**Comment by jmacwhyte** [comment]
I see it now! Thanks. I didn't know what to search for in the docs.
